hikari/api/event_dispatcher.py

```diff
@@ -54,74 +54,25 @@ class EventManager(abc.ABC):
 
     __slots__: typing.Sequence[str] = ()
 
-    @abc.abstractmethod
-    def add_raw_consumer(self, name: str, consumer: ConsumerT, /) -> None:
-        """Register a given async callback to a raw event name.
-
-        Parameters
-        ----------
-        name : str
-            The case-insensitive name this event should be triggered based on.
-        consumer : typing.Callable[[gateway_shard.GatewayShard, hikari.internal.data_binding.JSONObject], typing.Coroutine[typing.Any, typing.Any, None]]
-            The async function to invoke on each raw event. This should take
-            two positional arguments, the shard that this event is being
-            triggered by and this raw event's payload.
-        """  # noqa: E501 - Line too long
-
-    @abc.abstractmethod
-    def get_raw_consumers(self, name: str, /) -> typing.Sequence[ConsumerT]:
-        """Get the async callbacks registered for a raw event.
-
-        Parameters
-        ----------
-        name : str
-            The case-insensitive name of the event to remove a raw consumer for.
-
-        Returns
-        -------
-        typing.Sequence[typing.Callable[[gateway_shard.GatewayShard, hikari.internal.data_binding.JSONObject], typing.Coroutine[typing.Any, typing.Any, None]]]
-            A sequence of the found async functions registered for the provided
-            event name, this will be an empty sequence if no consumers were found.
-        """  # noqa: E501 - Line too long
-
-    @abc.abstractmethod
-    def remove_raw_consumer(self, name: str, consumer: ConsumerT, /) -> None:
-        """Remove a registered async raw event consumer callback.
-
-        Parameters
-        ----------
-        name : str
-            The case-insensitive name of the event to remove a raw consumer for.
-        consumer : typing.Callable[[gateway_shard.GatewayShard, hikari.internal.data_binding.JSONObject], typing.Coroutine[typing.Any, typing.Any, None]]
-            The async function to remove from the registered raw consumers.
-
-        Raises
-        ------
-        LookupError
-            If the either `name` or `consumer` couldn't be found in the
-            registered raw consumers.
-        """  # noqa: E501 - Line too long
-
     @abc.abstractmethod
     def consume_raw_event(
```

Comment:
Also sorry if this is annoying but if we're keeping the consume method on the interface would it be worth it keeping the interface called event manager?